src/Tizen.NUI/src/public/BaseComponents/CustomView.cs

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  *
  */
+using System;
```

Comment:
Do you really need this using System?

Reply:
Otherwise, I got a build error for the Obsolete keyword for deprecated API.